WSL/toc.yml

```diff
@@ -0,0 +1,50 @@
+- name: Overview
+  href: overview.md
+  items:
+    - name: About
+      href: about.md
+    - name: WSL1 and WSL2
+      href: wsl1-wsl2.md
+- name: Getting Started
+  href:
+  items:
+    - name: Install WSL1
+      href: install-win10.md
+    - name: Install WSL2
```

Comment:
Suggestion: Swap this page to be at the top and rename it to: `Install WSL`
This will help more users install WSL2! :)

Reply:
Also make sure that we use the `install-win10.md` to have the WSL2 install instructions, that way blogs that link to the `install-win10.md` page will automatically link to the new WSL2 install instructions!